app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_split = command.split()
+    if len(command_split) != 3 or "mv" not in command_split[0]:
```

Comment:
"mv" in "pleasemv" will return true
"mv" == "pleasemv" will return false
Check difference about == and in
```suggestion
    if len(command_split) != 3 or "mv" != command_split[0]:
```